benchmark/mnk_provider.py

```
@@ -0,0 +1,145 @@
+
+    if operation in {"swipe", "drag"}:
+        params = _params(payload, operation)
+        path = _path(params.get("path"))
+        _button(params.get("button", "left"))
+        return [
+            ("touch_gesture", {
+                "type": operation,
+                "start": {"x": start[0], "y": start[1]},
+                "end": {"x": end[0], "y": end[1]},
+            })
+            for start, end in zip(path, path[1:])
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)